corehq/apps/ota/migrations/0006_update_blob_paths.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-08-03 13:24
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import HqRunSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ota', '0005_rm_max_length_from_integer'),
+    ]
+
+    operations = [
+        HqRunSQL(
+            # This migration is not reversible because blobs created
+            # since the migration will no longer be accessible after
+            # reversing because the old blob db would use the wrong path
+            """
+            UPDATE ota_demouserrestore
+            SET restore_blob_id = '_default/' || restore_blob_id
```

Comment:
took me a while to figure out that `_default` was the bucket name under the old API. Adding that to the comment could help future readers.

Reply:
0b0d406f3f96f9667f75d227d8df70f1be1d0f2e